webpack.config.js

```diff
@@ -56,9 +56,10 @@ let config = [
       notification_list: './frontend/www/js/omegaup/notification/list.js',
       problem_edit: './frontend/www/js/omegaup/problem/edit.js',
       problem_feedback: './frontend/www/js/omegaup/problem/feedback.js',
-      problem_list: './frontend/www/js/omegaup/problem/list.js',
+      problem_list: './frontend/www/js/omegaup/problem/list.js',     
```

Comment:
nit:

```suggestion
      problem_list: './frontend/www/js/omegaup/problem/list.js',
```

hay que evitar dejar espacios al final de las líneas.

Reply:
listo